temporian/core/operators/propagate.py

```diff
@@ -104,55 +107,39 @@ def build_op_definition(cls) -> pb.OperatorDef:
 
 def propagate(
     event: Event,
-    to: Union[Event, str, List[str]],
+    sampling: Event,
 ) -> Event:
-    """Extends index and propagates feature values.
-
-    Extends the index of `event` over the features of `add_event`. Feature
-    values from `event` are duplicated over the new index. `add_event` can be a
-    string or list of string representing features in `event`, or an event.
+    """Propagates feature values over a larger index.
 
-    For example, suppose an index-less event sequence containing two features
-    "f_1" and "f_2", and containing 16 timestamps. Suppose "f_1" is a numerical
-    feature and "f_2" is a string feature with 4 unique values across the 16
-    timestamps. "Propagating" feature "f_1" over "f_2" will create an event
-    sequence containing feature "f_1" and indexed by feature "f_2". This event
-    sequence will contain 4 indexed time sequences each containing 16
-    timestamps.
+    Given "event" and "sampling" where "event" contains a super index of

```

Comment:
I think you meant `sub index` instead of `super index` here

Reply:
In the context of set theory, and given we talk about the ensemble of index levels, I think the "super set" is correct here.
We still need to create a proper glossary with our terminology (and make sure the doc follows it) :)